app/views/reasonableExcuseJourneys/other/UploadEvidencePage.scala.html

```diff
@@ -52,7 +52,7 @@ <h1 class="govuk-heading-l">@messages("otherReason.uploadEvidence.heading")</h1>
 
 
     @formHelper(action = postAction,
-        args = (Symbol("data-multi-file-upload-start-rows"), "1"), (Symbol("data-multi-file-upload-max-files"), "5"), (Symbol("id"), "multi-upload-form")) {
+        args = (Symbol("data-multi-file-upload-start-rows"), "1"), (Symbol("data-multi-file-upload-max-files"), "5"), (Symbol("id"), "multi-upload-form"), (Symbol("multiFileUploadUploadedFiles"), previousUploads)) {
```

Comment:
`multiFileUploadUploadedFiles` should be `data-multi-file-upload-uploaded-files`, when the JS calls dataset on the form it gets parsed into camel-cased and drops the `data-` prefix

Reply:
Fixed